### ISSUE TITLE
[front] do we want to add a video tutorial?

### ISSUE THREAD

**Issue by GresilleSiffle** [tournesol-app/tournesol#1690]
A video could explain the comparison flow more pedagogically than the current tips.

The video could show:
- how to use the extension to add videos to the rate-later list
- how to use the video selector to select videos from the rate-later list
- how to compare

**Comment by lfaucon** [comment]
The difficulty to produce, maintain and update makes me prefer that we don't make a video tutorial and instead stick to tutorials as well placed tooltips on the website.

**Comment by aidanjungo** [comment]
Personally I think it would be useful to have one, it would allow to show more things in less time which could convince a certain audience to try Tournesol. However it would be best to do it when we have a stable version of the comparison page that should not change too much after.

**Comment by GresilleSiffle** [comment]
> The difficulty to produce, maintain and update makes me prefer that we don't make a video tutorial and instead stick to tutorials as well placed tooltips on the website.

That's right, a video may become quickly obsolete. If I remember correctly this idea of video, or more complete tutorial, emerged because we wanted to explain a lot of things about Tournesol outside of the current tutorial to keep it short.

Instead of a video, what do you think about this alternative: a "help" page that can be scrolled from top to bottom, with drawing explaining how to compare, and explaining the different element of the comparison UI (and more). Contrary to the current tutorial, this page will be always accessible, can be browsed at the user's pace, and can contain information about different subject (like the Auto button, the collective goals, etc.).
